paper_io_sim.tex

```diff
@@ -378,7 +378,7 @@
     and append it to the active list.
     If the blocks are dirty, we move them independently to the active list, to preserve their entry time.
     Because the chunk and block sizes may be different, there are situations
-    where a block is not entirely read.
+    in a block is not entirely read.
```

Comment:
do you mean `in which`?

Reply:
I thought ok saying "in which" but then thought "were" was ok. So this is a typo.